x-pack/plugins/apm/public/store/selectors/chartSelectors.ts

```diff
@@ -278,7 +278,7 @@ function getAnomalyBoundariesSeries(data: Coordinate[]) {
     data,
     type: 'area',
     color: 'none',
-    areaColor: rgba(colors.apmBlue, 0.1)
+    areaColor: rgba(theme.euiColorVis1, 0.1)
```

Comment:
Please double check that this actually works. `euiColorVis1` is a rgb value so `areaColor` will become: `rgba('rgb(49, 133, 252)', 0.1)` (which _should_ work in the latest version of `polished` but better be certain)

Reply:
Looks like it works as expected 
<img width="820" alt="screenshot 2019-02-04 at 12 06 01" src="https://user-images.githubusercontent.com/4104278/52204745-4840c600-2875-11e9-8661-4fc2d54e6852.png">